server/models/CountryV2.ts

```diff
@@ -0,0 +1,20 @@
+import CountryRepositoryV2 from 'infra/database/CountryRepositoryV2';
```

Comment:
@dagmawig so what's the difficulty we have to create a new CountryV2.ts for this? Couldn't we just reuse the old one?

Reply:
Hi Dag,

Thanks for the PR! I agree with @dadiorchen, I believe there isn't a need to create a separate model and database functions. Instead, it is better to create a new interface `CountryV2` that will eventually include the new columns from the denormalized table. For now, interface `CountryV2` will be the same as interface `Country`. Then, inside `server/infra/database/CountryRepository.ts`, you can create a second class that inherits from `BaseRepository<CountryV2>` and include all that you wrote in `server/infra/database/CountryRepositoryV2.ts`.
You will then be able to export and define both versions of the database models in `server/models/Country.ts`.
I believe this way will make it easier to apply changes to both versions of functions and will keep a simpler directory. Your routes will then invoke the correct version functions from the same `Country` model.